### ISSUE TITLE
respect setting of N from yaml file when discretizing AR1

### ISSUE THREAD

**Issue by sglyon** [EconForge/Dolo.jl#68]
Right now we extract N, but throw it away...

This isn't an obvious fix because we don't currently have room to store it anywhere

**Comment by albop** [comment]
True: right now, we don't. But there is an easy way of changing it later.
```
dp = discretize(model.exogenous; N=2)
time_iteration(model, dp)
...
```

**Comment by albop** [comment]
Another thing is we aim to make the definition of the exogenous process independent from how it is discretized.
So 
```
!VAR1
   rho: 0.43
   Sigma: [[0.002]]
   N: 5
```
doesn't make sense anymore. The information should be contained somewhere in `options`.
